webui/src/lib/components/layout.tsx

```diff
@@ -1,19 +1,24 @@
-import React, {FC, useState} from "react";
+import React, {FC, useContext, useState} from "react";
```

Comment:
Lint is inconsistent in the WebUI code - 
I tried to avoid messing these, but in some files it was a bit too much.